### DISCUSSION TITLE
Access netbox via browser

### DISCUSSION THREAD

**Discussion by gustavols** [netbox-community/netbox#10592] [Q&A]
Hello

I performed the installation of netbox in a Vm and apparently everything went well, but I wanted to know where do I access the netbox, what is the address, I put the ip address of the Vm and it is loading the page for a long time.

**Comment by cpund** [comment]
@gustavols how did you install Netbox? Via Docker image or via manual installation? Regardless, it's likely you need to append :8000 to your VM's IP address.

If that's not it, did you use Apache or Nginx? What distro are you using on your VM? May need to open up ports 443 and port 80.

**Reply by gustavols** [reply]
1 - Manual Installation
2 - I tried to find port 8000 and it won't
3 - Apache
4 - 20.04.05

**Reply by gustavols** [reply]
I'll take a look at the doors

**Reply by gustavols** [reply]
![image](https://user-images.githubusercontent.com/55259166/194424913-d2ae9797-6b95-4e12-8ba9-11e4b8b7c5f4.png)

**Reply by cpund** [reply]
Okay, a couple of things. Make sure apache is running.
> apachectl status
Should show 'active (running)' next to Active:. Additionally, make sure you have enabled apache2/httpd (I don't remember what name the service goes by on Ubuntu). 
> sudo systemctl enable apache2
Are you running a firewall? UFW? 
> sudo ufw allow 443

Make sure you're trying to access netbox via https:// and not http://

**Reply by gustavols** [reply]
OK, I am going to try.

**Comment by joaolucasmacedo** [comment]
Dis you follow the official installation guide? Did you setup the Web Serve(Apache/Nginx?
There is a couple of videos on youtube, both mine and Jeremy Strech's, performing a fresh install.

**Reply by gustavols** [reply]
Opa João Lucas, agradeço, voce tem o link do video?

**Reply by joaolucasmacedo** [reply]
Please, follow the official installation guide at docs.netbox.dev, as I do.
Here is [my channel](https://www.youtube.com/channel/UCVNe76IIwPwjkMLkW7b626Q).

**Reply by gustavols** [reply]
Joao lucas quando vou realizar a instalação do apache ou nginx ele fica travado no comando de instalação, sabe oque pode ser?

**Comment by gustavols** [comment]
![image](https://user-images.githubusercontent.com/55259166/194618427-70771b31-2f04-443d-90b3-601dbce487b1.png)

**Reply by cpund** [reply]
I'd double check you're not running anything else on that server... I tried curling it and I got some message about Azure Bastion. Additionally, verify you're using the machine's actual IP address by looking at the interfaces with 'ip a'.
